delft/sequenceLabelling/wrapper.py

```diff
@@ -134,6 +134,14 @@ def __init__(self,
                                               early_stop, patience,
                                               max_checkpoints_to_keep, multiprocessing)
 
+        print("---")
+        print("max_epoch:", max_epoch)
+        print("early_stop:", early_stop)
+        print("batch_size:", batch_size)
+        print("max_sequence_length:", max_sequence_length)
+        print("model_name:", model_name)
+        print("---")
+
```

Comment:
I would put it in the train/eval method? Because these values are not the correct ones when the model will be then loaded, eg:

```
        model = Sequence(model_name)
        model.load()
```

It's only after `load()` than the config values are set.

When training from scratch, the config values could also be changed between the model creation and the actual training, although it's never the case in the current `applications/*` tasks.

Reply:
Where do you mean to move the `prints`? In the `application/*` ?

Reply:
Rather in the wrapper.train() and/or wrapper.eval(), because we know at this point that the configuration parameters are set (this is not the case when instantiating the architecture) and I guess that printing parameters is relevant only for train and eval.